test/micromamba/test.sh

```diff
@@ -7,4 +7,17 @@ source dev-container-features-test-lib
 
 check "default (latest) version" micromamba --version
 
+no_conda_forge() {
+    micromamba config list | (! grep -q conda-forge)
+}
+check "no-conda-forge" no_conda_forge
+
+micromamba install --yes --channel=conda-forge python
```

Comment:
I don't know why this command fails by non-root users...... (obviously I lack understanding of Conda and Mamba!)
Do we need additional commands to run micromamba non-interactively?

Reply:
The user needs to have an initialized shell, i.e. `micromamba activate`. We add this command to `~/.bashrc` [here](https://github.com/mamba-org/devcontainer-features/blob/1a36e997d10589b1a95d732737866c7d504b0ccd/src/micromamba/install.sh#L151). Unfortunately, if `~/.bashrc` doesn't run for whatever reason (shell type not correct, or the user changes), then the environment will not be active.

Reply:
Thank you for telling me about it.
However, since 7086d5a also failed, it seems that the `bashrc` is loaded correctly and is not the cause of the error.

Reply:
For non-root users, the packages option also seems to fail by a permission error...
https://github.com/mamba-org/devcontainer-features/actions/runs/3839733691/jobs/6537902861#step:4:2455
This could be a micromamba issue, but I don't know the details.

Reply:
This seems strange to me. Unfortunately I don't have any time to look at this now. :(

Reply:
The `~/conda-bld` seems to come from the channels config [here](https://github.com/mamba-org/mamba/blob/a94d5ac9d6450c6058d02247fe95f3175e2cab98/libmamba/src/core/channel.cpp#L822). I'm not sure what this means for the problem.

Reply:
Thank you for taking a look at this.
Fortunately these errors are irrelevant when installing this Feature and using micromamba interactively, so I think we can leave it as is for now.